decidim-comments/app/packs/src/decidim/comments/comments.js

```diff
@@ -1,17 +1,28 @@
 import CommentsComponent from "src/decidim/comments/comments.component"
+import { screens } from "tailwindcss/defaultTheme"
 
 window.Decidim.CommentsComponent = CommentsComponent;
 
 const commentsInitializer = () => {
+  const isMobileScreen = () => window.innerWidth < parseInt(screens.md, 10);
+
   // Mount comments component
   $("[data-decidim-comments]").each((_i, el) => {
     const $el = $(el);
     let comments = $(el).data("comments");
+    let wasMobileScreen = isMobileScreen();
     if (!comments) {
       comments = new CommentsComponent($el, $el.data("decidim-comments"));
     }
     comments.mountComponent();
     $(el).data("comments", comments);
+    window.addEventListener("resize", () => {
```

Comment:
for some reason, when resizing the screen, the comments are disapearing. 

[Screencast from 2024-12-12 21-13-54.webm](https://github.com/user-attachments/assets/f2e918f1-adf3-49a3-aca9-30569749441b)